content/pages/api-versioning-strategy/index.md

```diff
@@ -0,0 +1,231 @@
+
+If your request doesn't include a version, then the API defaults to the `V2` Box API version. However, we do not recommend relying on this behavior for adopting deprecated changes. As you update your app, you should specify the API version with every request. By making your app version-aware, you anchor your code to a specific set of features that are guaranteed to behave in the same way for the supported timeframe.
+
+## Calling an API version
+
+Box API versions are explicitly declared in the `Box-Version` header that your app makes requests to. For example:
+
+```sh
+curl --location 'https://api.box.com/2.0/sign_requests' \
+--header 'Box-Version: 2025.0' \
+--header 'Authorization: Bearer …
+```
+
+The client gets a list of all created sign requests and asks for version `2025.0`. There are several supported versions of the APIs available, and you specify the version that you want to use with the `Box-Version` header. There are three types of API versions: **stable**, **deprecated**, and **unstable**.
+
+Box API responses contain the header `Box-Version`, which returns the API version that was used to perform the request. When you keep your app updated, this matches the API version that's specified in your request. If the returned version is different, then your app is out of date and is using the default API version.
+
+## Versioning errors and fallback
+
+### Calling an incorrect API version in the URL
+
+Box documentation specifies API URLs. For instance, the Sign Requests endpoints are accessed via: `https://api.box.com/2.0/sign_requests/`. If a customer mistakenly makes a call to an incorrect version, such as `https://api.box.com/3.0/sign_requests/`, the response returns an `HTTP error code 404 - Not Found` error.
+
+### Calling a deprecated API
+
+When a customer uses an API version that Box has marked as deprecated, the API will respond as usual. However, it will append a `Deprecation` header, stating the deprecation date, for example:
+
+```sh
+Deprecation: date="Fri, 11 Nov 2018 23:59:59 GMT"
+```
+
+Customers should monitor API responses and take note when this header appears, signaling the need to plan for the transition to a new API version.
+
+### Calling a nonexistent version
+
+If a customer attempts to use an outdated API version, such as `2023.0`, which has reached its end-of-life, the response will return an `HTTP error code 404 - Not Found`. See [Calling an incorrect API version in the URL](#making-a-call-with-incorrect-api-version-in-the-url) for more information.
+
+## How Box SDK versioning works
+
+The versioning strategy is only applied to [next generation generated SDKs](https://developer.box.com/sdks-and-tools/).
```

Comment:
Please use the direct link https://developer.box.com/sdks-and-tools/#next-generation-sdks